CMakeLists.txt

```diff
@@ -47,22 +50,28 @@ if (OSX)
 endif ()
 
 
-# no unknown pragmas: ODB compiler uses some pragmas that the regular compiler doesn't need
-set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wno-unknown-pragmas")
-# Needed to statically link ODB
-set (CMAKE_C_FLAGS " -DSQLITE_ENABLE_UNLOCK_NOTIFY -DSQLITE_OMIT_LOAD_EXTENSION")
-set (CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
-if (NOT CMAKE_BUILD_TYPE)
-    set (CMAKE_BUILD_TYPE  "RelWithDebInfo" CACHE STRING "Choose the type of build, options are: None(CMAKE_CXX_FLAGS or CMAKE_C_FLAGS used) Debug Release RelWithDebInfo MinSizeRel." FORCE)
+if (WINDOWS)
+    include_directories (lib/sqlite)# windows specific
+    include_directories (inc/windows_specific)# windows specific
+    set (CMAKE_BUILD_TYPE  "Release" CACHE STRING "On windows we only support release mode" FORCE)
```

Comment:
Then what about the plan we suggested for Travis to run in "Debug" mode to make it faster?

Reply:
I will set it to `cache` instead of `force` once I confirm that debug will work fine.
then we can pass the argument `cmake -DCMAKE_BUILD_TYPE=Debug`